build.cmd

```diff
@@ -1,5 +1,7 @@
+set build_type=%1
 mkdir build
 cd build
-cmake ../
-MSBuild revengeMusic.sln
-cd ..
+mkdir %build_type%
```

Comment:
There's no need to separate the CMake generated files. CMake transfers the configurations to VS so you can build both configurations and VS will output them to Debug/ and Release/ receptively. If you change the script up a bit you could run `build Debug & build Release` or via MSBuild `msbuild revengeMusic.sln /p:Configuration=Debug & msbuild revengeMusic.sln /p:Configuration=Release`.